### PR TITLE
Update participant-agreement-spanish-no-compensation.md

### PR DESCRIPTION
**Pull request by allisonpress**
## Description of changes

> Describe what this pull request is changing. Try to use plain language as much
> as possible to help teammates understand what they are reviewing.

## Known issues

> If this pull request introduces anything that needs to be addressed later,
> note it here. Some examples might be if you introduced a new link that fails
> tests but will work when another pull request is merged; you created a new
> method placeholder but it does not yet have content; or your change breaks
> something else that will be removed later. If you create any issues to
> address issues created by this pull request, this is a good place to list
> them! You can delete this section if the PR does not introduce any new issues.

## Related issues

> If this pull request is associated with any other issues or pull requests,
> lis them here. If this PR *closes* an issue, list it as `closes #issue` so
> the issue will automatically be closed when the PR is merged. If there are no
> related issues or PRs, you can delete this section.

- 
- 

## Reviewers:

> In addition to assigning reviewers in the GitHub interface, please also list
> who you would like a review from here and, if applicable, what kind of review
> you would like from them. For example, you might ask one person to review
> content while someone else reviews visual layout.

- [ ] @username: please review content
- [ ]
- [ ]

### CODE REVIEW

File: _pages/resources/participant-agreement-spanish-no-compensation.md [hunk 1]
```diff
@@ -1,54 +1,48 @@
 ---
-title: Ejemplo del acuerdo para participantes en un estudio de diseño
-description: An example design research participant agreement in Spanish
+Spanish title: Contrato para participantes en el Estudio de Investigación (sin pagar)
+English title translation: Design research participant agreement (without compensation)
```
Comment (geekygirlsarah) [inline _pages/resources/participant-agreement-spanish-no-compensation.md:3]: In the front matter (area between the `---`), you have to keep the words before the colon the same. `title:` tells Jekyll how to title the page within the HTML, it's not regular content displayed on the page. You can edit the text after the colon.  The description should go back on it.

**Review by geekygirlsarah — CHANGES_REQUESTED** [review]
The front matter needs some rolling back, but the content seems fine (as someone who doesn't speak Spanish but fed some of it through DeepL's translator).

File: _pages/resources/participant-agreement-spanish-no-compensation.md (hunk 1 above)
Comment (bpdesigns) [reply _pages/resources/participant-agreement-spanish-no-compensation.md:3]: @geekygirlsarah @allisonpress @MelissaBraxton just checking in to see what the next steps are for this. 

If "Spanish" cannot be before the colon, how should we adjust the text?

Comment (MelissaBraxton) [reply _pages/resources/participant-agreement-spanish-no-compensation.md:3]: @geekygirlsarah  - Are you able to quickly make the edit your recommend? I'm not sure I'm following your comment, but if you want to make the edit, then Ben or I can review and merge.

Comment (bpdesigns) [reply _pages/resources/participant-agreement-spanish-no-compensation.md:3]: @geekygirlsarah just checking in to see if you are still able to work on this or if it should be reassigned. Thanks. cc/ @MelissaBraxton

Comment (geekygirlsarah) [reply _pages/resources/participant-agreement-spanish-no-compensation.md:3]:
```suggestion
title: Ejemplo del acuerdo para participantes en un estudio de diseño
description: Design research participant agreement (without compensation)
```

Comment (MelissaBraxton) [reply _pages/resources/participant-agreement-spanish-no-compensation.md:3]: Ah, ok! That makes sense now. I can remove the extraneous words before the colon, but I'll keep the other language the same, as we had it translated by a Spanish speaker.

**Review by MelissaBraxton — APPROVED** [review]

**Comment by MelissaBraxton** [comment]
LGTM!

**Comment by geekygirlsarah** [comment]
This is failing for the known issue #394, and doesn't really hinder this getting published. I think it can be merged if you're ok with it @MelissaBraxton

**Review by geekygirlsarah — APPROVED** [review]

**Review by bpdesigns — APPROVED** [review]

**Pull request merged**